dir.c

```diff
@@ -906,18 +912,24 @@ dir_tell(VALUE dir)
 
 #ifdef HAVE_SEEKDIR
 /*
- *  call-seq:
- *     dir.seek( integer ) -> dir
+ * call-seq:
+ *   seek(position) -> self
  *
- *  Seeks to a particular location in <em>dir</em>. <i>integer</i>
- *  must be a value returned by Dir#tell.
+ * Sets the position in +self+ and returns +self+.
+ * The value of +position+ should have been returned from an earlier call to #telldir;
+ * if not, the return values from subsequent calls to #readdir are unspecified.
```

Comment:
It's `telldir` and `readdir` in the system API, which are the `Dir#tell` and `Dir#read` methods in Ruby.

```suggestion
 * The value of +position+ should have been returned from an earlier call to #tell;
 * if not, the return values from subsequent calls to #read are unspecified.
```

Reply:
Fixed.